src/constants/tokens.ts

```diff
@@ -94,7 +94,8 @@ export const WETH_TOKEN = new Token({
   decimals: 18,
   addresses: WETH_ADDRESSES,
   factory: IERC20__factory,
-  purchaseUrl: "",
+  purchaseUrl: `https://etherscan.io/token/${WETH_ADDRESSES[NetworkId.MAINNET]}`,
+  explorer: true,
```

Comment:
I don't know if I like this logic? On a traditional bond with WETH or DAI being the bondable asset (rather than the payout) you would need to know where to purchase them, but this logic would show you the etherscan url... the only reason I brought up etherscan was specifically for Inverse Bonds they are the payout token, so in that case you don't need to know where to get them. what do you think?

Reply:
these purchaseUrls were previously empty, so didn't seem like they would go anywhere (why you were seeing the behavior you did). but you're right in that once regular bonds turn on, it'd link to an 'explorer' vs. an empty link. 

What if we add both an explorerURL and a purchaseURL here instead of an explorer boolean? 
This would give us the ability to handle both cases well. If the explorerURL is set, and it's an inverse bond we'll display the explorer url instead of the purchase url, and vice versa.

Reply:
I think that makes sense.